src/Compilers/CSharp/Test/Semantic/Semantics/RefFieldTests.cs

```diff
@@ -166,15 +184,24 @@ public void RefField(bool useCompilationReference)
     public ref T F;
     public S(ref T t) { F = ref t; }
 }";
-            var comp = CreateCompilation(sourceA, parseOptions: TestOptions.Regular10);
+            // Avoid sharing mscorlib symbols with other tests since we are about to change
+            // RuntimeSupportsByRefFields property for it.
+            var mscorlibRefWithRefFields =
+                ((AssemblyMetadata)((MetadataImageReference)MscorlibRef).GetMetadata()).CopyWithoutSharingCachedSymbols().
+                     GetReference(display: "mscorlib.v4_0_30319.dll");
```

Comment:
Can we calculate this once and cache the value, perhaps in CSharpTestBase?

Reply:
Or consider extracting a helper method if each caller needs a distinct instance.